pages/1.11/release-notes/1.11.0-rc4/index.md

```diff
@@ -38,7 +38,7 @@ Please try out the new features and updated data services. Provide any feedback
 
 - DC/OS 1.11.0 is integrated with the latest 1.6 release of Marathon. For more information about Marathon 1.6, consult the [Marathon changelog](https://github.com/mesosphere/marathon/blob/master/changelog.md).
 
-- DC/OS 1.11.0 supports latest Kubernetes 1.9 Container Scheduler. For more information about Kubernetes 1.0 on DC/OS, consult the [Kubernetes on DC/OS changelog](https://docs.mesosphere.com/services/beta-kubernetes/0.4.0-1.9.0-beta).
+- DC/OS 1.11.0 supports latest Kubernetes 1.9 Container Scheduler. For more information about Kubernetes 1.0 on DC/OS, consult the [Kubernetes on DC/OS changelog](https://docs.mesosphere.com/services/beta-kubernetes/0.6.0-1.9.1-beta).
```

Comment:
Link will be https://docs.mesosphere.com/services/kubernetes/1.0.0-1.9.3, once latest docs are merged.